lib/worker.go

```diff
@@ -307,8 +305,8 @@ func (w *DefaultBaseWorker) MetaKVClient() metaclient.KVClient {
 	return w.userMetaKVClient
 }
 
-func (w *DefaultBaseWorker) UpdateStatus(ctx context.Context, status WorkerStatus) error {
-	err := w.statusSender.AsyncSendStatus(ctx, status)
+func (w *DefaultBaseWorker) UpdateStatus(ctx context.Context, status libModel.WorkerStatus) error {
+	err := w.statusSender.UpdateStatus(ctx, &status)
```

Comment:
Please add comment about the rate limit and delivery guarantee of `UpdateStatus`.
If worker sends messages with higher frequency than the processing rate than master, the overflow messages will be dropped in master side, is it right